### DISCUSSION TITLE
Welcome to claasp Discussions!

### DISCUSSION THREAD

**Discussion by joc7188** [Crypto-TII/claasp#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by aetheris04** [comment]
After running the command make rundocker 
The following error is coming ERROR [ 3/51] RUN apt-get install --no-install-recommends -y gfortran   gcc   g++   lib

**Reply by AnaCaceres** [reply]
Hi ez-tahseen! 
Is this still happening? If this is the case, could you provide us more information about the error in order for us to be able to help you?
Thank you.

**Reply by aetheris04** [reply]
No, now it's working fine. 
But can I have any template code for adding more ciphers that are not already included in the database?

**Reply by AnaCaceres** [reply]
There is an explanation of the ciphers specifications in **_/docs/CIPHER.md_**, you can take any cipher inside **_/claasp/ciphers_** as a template to create you own cipher. Take into consideration that all of the ciphers need to extend from **_/claasp/cipher.py_** file, you can see how it is done in any of the ciphers that were just mentioned. 
Hope this helps.
